.github/workflows/ci_eval_short.yaml

```diff
@@ -69,4 +69,4 @@ jobs:
       - name: Run perplexity test with vmfb
         run: |
           source ${VENV_DIR}/bin/activate
-          pytest -n 8 -v -s sharktank/tests/evaluate/perplexity_iree_test.py --bs=5 --iree-device='hip://6' --iree-hip-target=gfx942 --iree-hal-target-backends=rocm --llama3-8b-f16-model-path=/data/llama3.1/8b/llama8b_f16.irpa --llama3-8b-tokenizer-path=/data/llama3.1/8b/tokenizer_config.json
+          pytest -n 8 -v -s sharktank/tests/evaluate/perplexity_iree_test.py --bs=5 --iree-device=hip --iree-hip-target=gfx942 --iree-hal-target-backends=rocm --llama3-8b-f16-model-path=/data/llama3.1/8b/llama8b_f16.irpa --llama3-8b-tokenizer-path=/data/llama3.1/8b/tokenizer_config.json
```

Comment:
Not sure what to make of this segfault: https://github.com/nod-ai/shark-ai/actions/runs/12439587223/job/34733891111?pr=725#step:6:42 . Looks preexisting? Do we have that tracked anywhere? Has anyone been looking into it?

Reply:
A [previous](https://github.com/nod-ai/shark-ai/actions/runs/12423222212/job/34712023210#step:5:383) CI run passed with the same IREE version and able to repro issue locally.  Will work on a separate PR to fix this.